src/stores/SegmentFeatureStore.ts

```diff
@@ -192,6 +193,9 @@ export class SegmentFeatureStore {
         imageSetStore: ImageSetStore,
         checkOverwrite: boolean,
         overwriteFeatures: boolean,
+        featuresToCalculate: string[],
+        // todo - add args (features to calc) here
+        // markerStats: PlotStatistic[]
```

Comment:
Should featuresToCalculate be PlotStatistic[] instead of string[]? Can probably get rid of the comments now.

Reply:
got it working, not sure why i had an issue before.